### DISCUSSION TITLE
Slint 1.3 Released with Revamped Native Styles and JavaScript API

### DISCUSSION THREAD

**Discussion by ogoffart** [slint-ui/slint#3897] [Blog Comments]
[Blog post](https://slint.dev/blog/slint-1.3-released)

[ChangeLog](https://github.com/slint-ui/slint/blob/master/CHANGELOG.md#130---2023-11-10)

<hr /><em>This discussion was created from the release <a href='https://github.com/slint-ui/slint/releases/tag/v1.3.0'>1.3.0</a>.</em>

**Comment by adrian5** [comment]
Awesome work guys! Solid progress as always.
